server/sdk/types/ITeamService.ts

```diff
@@ -7,9 +8,12 @@ export interface ITeamCreateParams {
 	members?: Array<string>; // list of user _ids
 	owner?: string; // the team owner. If not present, owner = requester
 }
-
 export interface ITeamService {
 	create(uid: string, params: ITeamCreateParams): Promise<ITeam>;
+	addRoom(uid: string, rid: string, teamId: string, isDefault: boolean): Promise<IRoom>;
+	removeRoom(uid: string, rid: string, teamId: string): Promise<IRoom>;
+	listRooms(uid: string, teamId: string): Promise<Array<IRoom>>;
```

Comment:
please use a return can be used for pagination (like `list` and `listAll`)
```suggestion
	listRooms(uid: string, teamId: string): Promise<IRecordsWithTotal<IRoom>>;
```